.github/workflows/build_and_test.yml

```diff
@@ -565,9 +581,31 @@ jobs:
         docker build -t $IMAGE_NAME:$DOCKER_TAG .
         docker push $IMAGE_NAME:$DOCKER_TAG
 
-        # build and push the broader alias tag
-        docker tag $IMAGE_NAME:$DOCKER_TAG $IMAGE_NAME:$RELEASE
-        docker push $IMAGE_NAME:$RELEASE
+        # is_gt() {
+        #   local v1=(${1//./ })
+        #   local v2=(${2//./ })
+
+        #   for i in $(seq 0 2); do
+        #     (( ${v1[$i]} < ${v2[$i]} )) && return 1
+        #     (( ${v1[$i]} > ${v2[$i]} )) && return 0
+        #   done
+        # }
+
+        if [[ $RELEASE == 'latest' ]]; then
+          # https://gist.github.com/sampaiodiego/3992a07fdc4e43fd776acd7f36821ea1
+          CURRENT_LATEST="$(
+            git -c 'versionsort.suffix=-' ls-remote -t --exit-code --refs --sort=-v:refname https://github.com/RocketChat/Rocket.Chat '*' |
```

Comment:
do you know if there is an env var or something else we can use instead of having the repo's url hardcoded here?

Reply:
`GITHUB_REPOSITORY` should do the trick